hack/update-cpi-chart.sh

```diff
@@ -9,8 +9,8 @@ chart_dir="./helm/cloud-provider-vsphere/charts/cloud-provider-for-vsphere"
 cd "$base_dir"
 
 "./hack/clone-git-repo.sh" \
-    "giantswarm/cloud-provider-vsphere" \
-    "improve-helm-chart" \
+    "/kubernetes/cloud-provider-vsphere" \
```

Comment:
@erkanerol's changes were merged upstream, I don't think we need to use our fork anymore.